transport/src/main/java/io/scalecube/transport/TransportConfig.java

```diff
@@ -12,6 +12,7 @@
 
   public static final TransportConfig DEFAULT = builder().build();
 
+  public static final boolean DEFAULT_PREFER_IP6 = false;
```

Comment:
Lets for consistency add here defaults for listen address/interface too with null (or empty string) values. You can add comment that this means fallback to getLocalHost.